ultimatepython/classes/encapsulation.py

```diff
@@ -55,12 +64,12 @@ def __set_account_number(self, number):
         """
         self.__account_number = number
 
-    def delete_account(self):
+    def remove_account_details(self):
         """
         This method is used to delete an account.
         """
-        self.__balance = 0.0
-        self.__set_account_number(0)
+        self.__balance = 0
+        self.__set_account_number(0)  # calling the private method within the class.
```

Comment:
Correction: You're calling this logic within the public instance method. But why not add this in the docstring anyways?

Reply:
Fixed.